main.py

```diff
@@ -126,10 +116,19 @@ async def submit_experiment(
         priority: Higher value means sooner scheduling.
         timed: The due date for the experiment in ISO format.
           None for no due date.
+        visualize: If True, the experiment file is modified for visualization.
+          The original file and vcd file are saved in the visualize path set in config.json.
     
     Returns:
         The run identifier, an integer which is incremented at each experiment submission.
     """
+    if visualize:
+        experiment_path = posixpath.join(configs["master_path"], configs["repository_path"], file)
+        with open(experiment_path, encoding="utf-8") as experiment_file:
+            _code = experiment_file.read()
+        # TODO(BECATRUE): The code will be modifed in #37.
+    else:
+        pass
```

Comment:
Is this necessary? BTW why doesn't pylint complain about this?

Reply:
This is necessary later becasue I will set the value of `file` in `expid` as a variable, and assign the appropriate path if the visualize option is `True` or not.

Reply:
> BTW why doesn't pylint complain about this?

I think if there is no `pass`, it will cause a syntax error, so in this case, Pylint seems to allow it.

Reply:
Okay then please add a TODO comment, since `else: pass` itself is a meaningless statement.